### PR TITLE
feat: Added balance sheet to menu

### LINKED ISSUE #672: feat: add balance sheet to menu
# Problem
For our NPO, we need to display our balance sheet publicly.  Let's add it to the hamburger menu

### PR DESCRIPTION
**Pull request by ermish**
Resolves #672

## 🔧 What changed
- Added balance sheet to hamburger menu

## 📸 Screenshots

-   ### Before
image

-   ### After
image

### CODE REVIEW

File: i18n/locales/en.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
     },
     "hamburgerMenu": {
         "about": "About Find a Doc, Japan",
+        "balancesheet": "Balance Sheet",
```
Comment (NabbeunNabi) [inline i18n/locales/en.json:20]: Would be cool if you could automate adding this to the other files

File: components/HamburgerMenu.vue [hunk 2]
```diff
@@ -183,14 +183,28 @@
...
+                                    target="_blank"
+                                    class="underline"
+                                >#9011005010215</NuxtLink>
+                            </span>
+                            <span class="ml-2 mt-0.5">
+                                <NuxtLink
+                                    :to="'https://www.npo-hiroba.or.jp/search/zoom.php?pk=121289'"
+                                    target="_blank"
+                                    class="underline"
+                                >{{ $t('hamburgerMenu.balancesheet') }}</NuxtLink>
```
Comment (NabbeunNabi) [inline components/HamburgerMenu.vue:206]: Our linter didn't want the closing tag on a separate line? 🤔 Interesting

Comment (ermish) [reply components/HamburgerMenu.vue:206]: nope 🤔 but I'll move it

File: i18n/locales/en.json (hunk 1 above)
Comment (ermish) [reply i18n/locales/en.json:20]: haha, if only there was a way.....

**Pull request merged**